### DISCUSSION TITLE
Fragen zu Solar‐Passthrough und einem 2. Display

### DISCUSSION THREAD

**Discussion by jpbuxde** [hoylabs/OpenDTU-OnBattery#1076] [Q&A] [answered]
Moin an Alle,

ich bin begeisterter DIY-Solar-Batterie Anhänger und von dem Projekt DTUOB begeistert .- vielen Dank -
Nun trotz vielen Lesens ergeben sich für mich doch noch einige Fragen.
Meine Anlage(n)
3x2 PV Module ,3x Victron 100/30, 2x 24V Batterie, 2x Hoymiles (1xHM300 , 1xHM600) 2x DTUOB (über verschiedene IP-Adressen)
Meine Fragen: 1.
ich habe z.Zt 2Victron die eine Batterie versorgen und ein HM600 der rund um die Uhr aus dieser Batterie zieht. Diese Anlage wird über DPL (Shelly3EMpro) geregelt. Die 2. Anlage hat einen Victron ->Batterie->HM300 bei diesem ist DLP ausgeschaltet und nur eine Grundlast mit der rund um die Uhr eingespeist wird.
Ist diese Konfi OHNE das Solar‐Passthrough dazugeschaltet ist nachteilig? Da ich lese das die WR während der Ladephase der Batterie im Regelfall aus sind bin ich mir nicht sicher. Offensichtlich funktioniert bei mir z.Zt alles. Die Batterien werden an Tag geladen ,wenn sie voll sind gehen die Victrons auf Float und die Solarleistung wird weggeregelt.
Wo wäre der Vorteil von Solar‐Passthrough?
2. Da ich die DTUs in der Nähe der Batterien/JKBMS platzieren muß (JK und Victron Einbindung)ist das Display ,welches ich sehr schätze,nicht sichtbar(andere Etage im Haus). kann ich eine 2. DTUOB bei der ich alle Regelungen ausschalte quasi nur zum sehen unter der IP parallel mitlaufen lassen?
Einige Hinweise sage "nein" weil die DTUs sich gegenseitig beeinflussen, aber wenn die 2. nur empfängt....?
Wenn das nicht möglich sein sollte, gibt es einen anderen gangbaren Weg?
Danke im Voraus
Jürgen

**Comment by spcqike** [comment]
> kann ich eine 2. DTUOB bei der ich alle Regelungen ausschalte quasi nur zum sehen unter der IP parallel mitlaufen lassen?

würde ich nicht machen. Meines Wissens nach stört sich das trotzdem und die Datenübertragung wird instabiler. Es könnte sogar passieren, dass die oDTUoB, die den DPL steuert, weniger gültige Daten bekommt und träger wird bzw abschaltet.

guck dir mal diesen Beitrag an
https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/965

Ich würd mir an deiner Stelle einen ESP mit Tasmota flashen und ein Display erstellen, wie du es brauchst. Du kannst alle nötigen Infos entweder per HTTP abholen lassen oder besser, per MQTT "lauschen" und einfach anzeigen.

**Comment by jpbuxde** [comment]
Danke für die Antwort,
das mit dem Display ist erstmal hintenan.
Solar‐Passthrough habe ich aktiviert. Wie es funktioniert soll sehe ich. Ist der Vorteil das die DTU den WR nicht so oft stellen muss wenn Solar‐Passthrough in Betrieb ist und der WR voll mit dem Überschuss in der Einspeisung läuft?
Da ich 2 DTUOB an 2 Batterien mit je einem WR betreibe funken auch 2 NRFs nebeneinander. Sie sollen zwar unterschiedliche WRs steuern aber kann es sein das sie sich trotzdem gegenseitig stören? Mir ist aufgefallen das mal die eine und mal die andere Funkverbindung zum WR abreißt (meistens von der die mit dem Stromzähler arbeitet). (WR-Feld wird rot) und nur durch Neustart oder kurze Stromtrennung wiederbelebt werden kann.

**Reply by spcqike** [reply]
Zum solar passthrough kann ich nicht viel sagen, da ich es selber nicht verwende. Guck doch einfach mal im Wiki. 
(Ich glaube aber dass es vornehmlich dazu da ist, verfügbare Solar Leistung direkt zu nutzen, wenn benötigt. Also dass nicht erst der Akku auf bspw 80% geladen wird und dann aus dem Akku entladen wird, was ja doppelte Verluste bedeutet. Mit S-Pt sollte der WR so viel produzieren wie solarleistung da ist bzw wie viel benötigt wird. Liegt mehr Solar an, geht der Überschuss in den Akku. Liegt weniger an, limitiert die solarleistung die Einspeisung und es wird nicht aus dem Akku entladen. Solange bis die Batterie start werte erreicht sind, dann wird auch aus dem Akku bedient. Oder nachts. Je nach Einstellung ?) 

zu den 2 Modulen:
Ja, die stören sich. Aber das tut das WLAN genauso. Dafür gibt es geteilte Bänder und Frequenzen und die Geräte gucken, ob der Kanal frei ist, und senden dann in einem freien Slot. Ist der Kanal voll, gibt es Probleme. Aber das sollte hier nicht der Fall sein. Zumindest kann ich mir das nicht vorstellen.

**Comment by jpbuxde** [comment, marked as answer]
@spcqike , ja du hast recht .Wenn die Batterie noch nicht die Entladestartspannung erreicht hat geht das vom Haus benötigte quasi vorbei. 
Bei mir ist es zur Zeit so das die DTUOB alle paar Stunden abstürzt. Entweder ändert sich die Leistung nicht mehr oder sie geht OFFLINE oder die IP ist gar nicht mehr erreichbar.
Helfen tut dann, wenn noch erreichbar, ein Neustart der DTUOB oder wenn gar nicht mehr erreichbar eine kurze Stromtrennung. Dann ist 30s später wieder alles io.

**Reply by jpbuxde** [reply]
Das schwächelnde WLAN ist zumindest eine Ursache gewesen.War zu sehr abgeschirmt durch den Schrank.Läuft jetzt nachdem ich es aus dem Schrank genommen habe seit 36h durch.
